src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name,
+    age,
+    sex,
+    isMarried,
+    partnerName } = person;
+
+  const maritalStatus = () => {
+    let status = '';
+
+    if (!isMarried) {
```

Comment:
There's a lot of unnecessary code here because it doesn't read well and pretty tough to maintain

You can check that if the person is unmarried, it returns "not married"
Otherwise you check with a ternary operator for the gender of the person and return the correct partner status with the name of the partner